addons/google_recaptcha/models/ir_http.py

```diff
@@ -24,8 +25,12 @@ def get_frontend_session_info(self):
 
     @api.model
     def _add_public_key_to_session_info(self, session_info):
-        """Add the ReCaptcha public key to the given session_info object"""
-        public_key = self.env['ir.config_parameter'].sudo().get_param('recaptcha_public_key')
+        """Add the ReCaptcha configuration to the given session_info object"""
+        config_params = request.env['ir.config_parameter'].sudo()
+        session_info['recaptcha_enabled'] = str2bool(
+            config_params.get_param('enable_recaptcha', default=True)
+        )
+        public_key = config_params.get_param('recaptcha_public_key')
```

Comment:
Is there value to have both data in the session info? Would not this be as good but with less session data to load and less JS code if we were to just not send the existing key in the session if recaptcha is not enabled?

Reply:
Yes seems better with one value. Actually, it's the same behaviour when the public key is not available: the script won't be loaded… So why not making this controlled by the setting value in backend and only use the "public key" check :+1:)